index.html

```diff
@@ -70,7 +70,7 @@ <h2>Tonic</h2>
           </div>
         </div>
         <div class="tonic">
-          <img src="images/story.png" alt="snap" />
+          <img src="images/Snapshoot-Portfolio-2.svg" alt="snap" />
```

Comment:
Kindly update the alt text so that it conveys the purpose of the respective image as described above 😃